src/PhpDoc/PhpDocNodeResolver.php

```diff
@@ -524,6 +525,24 @@ public function resolveRequireImplementsTags(PhpDocNode $phpDocNode, NameScope $
 		return $resolved;
 	}
 
+	/**
+	 * @return array<SealedTypeTag>
+	 */
+	public function resolveSealedTags(PhpDocNode $phpDocNode, NameScope $nameScope): array
+	{
+		$resolved = [];
+
+		foreach (['@psalm-inheritors', '@phpstan-sealed'] as $tagName) {
```

Comment:
I would go with `phpstan-inheritors` as psalm did. The reason is "sealed" term is too broad and can be easily confused with sealed array.

Also I think "inheritors" is a better name as we specify what direct child class can implement it, but we do not restrict further extending, ie. we do not seal it per se.

Reply:
This has already be decided on the phpdoc-parser library pr by ondrej

Reply:
See https://github.com/phpstan/phpdoc-parser/pull/273